logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnParticipantListChanged.kt

```diff
@@ -48,7 +59,6 @@ class OnParticipantListChanged(
             }
 
             callRepository.updateCallParticipants(
-                //TODO should be handled properly after supporting federated calls
                 conversationId = remoteConversationIdString.toConversationId().toString(),
```

Comment:
Since we have a mapper for federationId, could move all extension functions in QualifiedID to that class ?

Reply:
I'll prefer to leave it there, since the moving the logic, will imply to touch many classes. Maybe we can address that in other PR, when needed :D